float64[6] xyzypr
